src/redux/contacts/contacts.actions.js

```diff
@@ -0,0 +1,17 @@
+import { GET_CONTACTS, SET_CONTACTS, SET_LOADING } from './contacts.types';
+
+const setContacts = (contact) => ({
+  type: SET_CONTACTS,
+  payload: contact
+});
+
+const getContacts = () => ({
+  type: GET_CONTACTS
+});
+
+const setLoading = (loading) => ({
+  type: SET_LOADING,
```

Comment:
Please, rename action to the `SET_CONTACTS_LOADING` to reflect that it is loading for the contacts